rust/feature-flags/src/flags/flag_matching.rs

```diff
@@ -251,7 +286,7 @@ impl FeatureFlagMatcher {
             group_type_mapping_cache: group_type_mapping_cache
                 .unwrap_or_else(|| GroupTypeMappingCache::new(project_id, reader.clone())),
             groups: groups.unwrap_or_default(),
-            properties_cache: PropertiesCache::default(),
+            flag_evaluation_state: FlagEvaluationState::default(),
```

Comment:
this is more appropriately a `FlagEvaluationState`, since it addition to containing the relevant properties, it also contains evaluation results from the static cohort check.